src/Native/Runtime/EHHelpers.cpp

```diff
@@ -412,6 +414,9 @@ static UIntNative UnwindWriteBarrierToCaller(
     pContext->SetSp(sp+sizeof(UIntNative)); // pop the stack
 #elif defined(_ARM_) || defined(_ARM64_)
     UIntNative adjustedFaultingIP = pContext->GetLr();
+#elif defined(_WASM_)
```

Comment:
I would change this to:

```
#else
    UIntNative adjustedFaultingIP = 0;
    PORTABILITY_ASSERT("UnwindWriteBarrierToCaller");
#endif
```

so that it compiles fine for new ports.

Reply:
Done